dev_guide/deployments.adoc

```diff
@@ -191,6 +191,53 @@ To see what the rollback would look like without performing the rollback:
 $ oc rollback <deployment_config> --dry-run
 ----
 
+[[executing-commands-inside-a-container-deployments]]
+== Executing Commands Inside a Container
+
+You can add a command to a container, which modifies the container's startup
+behavior by overruling the image's `ENTRYPOINT`. This is different from a
+xref:../dev_guide/deployments.html#pod-based-lifecycle-hook[lifecycle hook],
+which instead can be run once per deployment at a specified time.
+
+Add the `*command*` parameters to the `*spec*` field of the deployment
+configuration. You can also add an `*args*` field, which modifies the
+`*command*` (or the `ENTRYPOINT` if `*command*` does not exist).
+
+====
+----
+...
+spec:
+  containers:
+    -
+    name: <container_name>
+    image: 'image'
+    command:
+      - '<command>'
+    args:
+      - '<arguments>'
```

Comment:
Having `<arguments>` (plural) as the single list element might lead the reader to put all the arguments on one line.  Maybe we could reformulate the `args` value to be a list of three elements: `<argument_1>`, `<argument_2>`, and `...`.